assets/xml/scenes/dungeons/bdan.xml

```diff
@@ -1,6 +1,7 @@
 <Root>
     <File Name="bdan_scene" Segment="2">
         <Cutscene Name="gJabuJabuIntroCs" Offset="0x155E0"/>
+        <Cutscene Name="gJabuJabuSapphireRetrieveUnusedCs" Offset="0x13080"/>
         <Scene Name="bdan_scene" Offset="0x0"/>
```

Comment:
personally I wouldnt mind if it was just "Jabu" instead of "JabuJabu", but I understand that something else is already named like that alread